erpnext_shopify/erpnext_shopify/doctype/shopify_settings/shopify_settings.py

```diff
@@ -67,7 +68,12 @@ def sync_shopify():
 			update_item_stock_qty()
 
 		except ShopifyError:
-			frappe.db.set_value("Shopify Settings", None, "enable_shopify", 0)
+			disable_shopify()
+			
+		except requests.exceptions.HTTPError, e:
+			if e.args[0] and e.args[0].startswith("402"):
+				disable_shopify()
+				sendmail_to_intimate_sync_disabled(_(""" Your shopify account has been disabled. Payment Required."""))
```

Comment:
- No need to have a function `sendmail_to_intimate_sync_disabled`.
- Also, there's an empty space at the start in the message.
- _("Shopify has suspended your account till you complete the payment. We have disabled ERPNext Shopify Sync. Please enable it once your complete the payment at Shopify.")